src/add-remove.js

```diff
@@ -0,0 +1,42 @@
+export default class Tasks {
+  constructor() {
+    this.tasksToDo = JSON.parse(localStorage.getItem('storedTask')) || [];
+    this.removeTask();
+    this.saveTask = this.saveTask.bind(this);
+  }
+
+  addTask(task) {
+    this.tasksToDo.push(task);
+    localStorage.setItem('storedTask', JSON.stringify(this.tasksToDo));
+  }
+
+  removeTask() {
+    for (let i = 0; i < this.tasksToDo.length; i += 1) {
+      if (this.tasksToDo[i].completed) {
+        this.tasksToDo.splice(i, 1);
+      }
+    }
+
+    this.tasksToDo.forEach((task) => {
+      task.index = this.tasksToDo.indexOf(task);
+    });
+
+    localStorage.setItem('storedTask', JSON.stringify(this.tasksToDo));
+  }
```

Comment:
## GREAT WORK SO FAR 👏

- [ ] All of the functionalities are working well already however there is still one to fix. I can not delete the tasks. You can add a delete icon next to the *three-dot* icon to make it easy to use 👍

![image](https://user-images.githubusercontent.com/69011963/154512290-a0b53d2b-b457-4754-a9cf-4764527cef17.png)